### PR TITLE
Using Statiq processes for tailwind commands

### PR DESCRIPTION
**Pull request by phil-scott-78**
Has @daveaglick's changes, and also a few other changes to get it working with tailwind's jit mode.

Since it's a two part PR with commits from two people might make sense to not squash this one.

### CODE REVIEW

**Comment by phil-scott-78** [comment]
I pulled down the generated docs and they look good. Appears I didn't break anything!

**Comment by daveaglick** [comment]
Nice!

File: docs/README.md [hunk 1]
```diff
@@ -10,15 +10,11 @@ The documentation site uses [Statiq](https://statiq.dev), a static site generato
...
 
-**Note that the site runs under a virtual directory.**
+## Npm
 
-## Building the CSS
-
-The documentation uses tailwind for the CSS. Committed to the repository is a large 4gb file with all possible styles at `assets/style.css`. This file is generated from the root files tailwind.config.js and tailwind.css. Any changes to the overall design style (e.g. colors, markdown styling, etc) can be made in these two files.
-
- For production, `npm run build:tailwind` should be ran post Statiq build. This process will look at all the styles in the HTML and prune the tailwind file down to a much more manageable 30kb or so file.
+The site uses some tools from the JavaScript ecosystem including npm. While Statiq will execute `npm install` and other commands as needed, you need to have [npm installed](https://www.npmjs.com/get-npm) before running a site build.
```
Comment (daveaglick) [inline docs/README.md:17]: Nice description, short and clear

**Review by daveaglick — APPROVED** [review]

**Review by patriksvensson — APPROVED** [review]

**Comment by patriksvensson** [comment]
Merged! 👍

**Comment by phil-scott-78** [comment]
hey @daveaglick - things went squirrelly on deploy. Looks like the CSS build happened after deploy. Where's the best time to get this running after the html has been built but before deploy?

build logs if you are curious - https://github.com/spectreconsole/spectre.console/runs/2673224486?check_suite_focus=true

**Comment by phil-scott-78** [comment]
Finally got a chance to look at the statiq config. I'm guessing if I want something to run before deployment I should go with `BeforeDeployment`. 

Gonna change that and resubmit

**Pull request merged**